ObligatorioDevOps3M/BackendProductsServiceExample
# Release (#9)

* update Docker image tag in GitHub Actions workflow

* add API products testing workflow and Postman collection

* update api products testing workflow to include feature branches

* remove Docker container run step from API products testing workflow

* update path for Postman collection in API products testing workflow

* update path for Postman collection in API products testing workflow

* update path for Postman collection in API products testing workflow

* update path for Postman collection in API products testing workflow

* update Postman collection path in API products testing workflow

* update path for Postman collection in API products testing workflow

* enhance API products testing workflow with Docker container management and Newman tests

* simplify Dockerfile CMD by removing redundant arguments

* set server port to 8080 in application properties

* replace api products testing workflow with api orders testing workflow

* Delete .github/workflows/maven.yml

* Feature/deploy eks (#7)

* add deploy to eks in workflow

* add deployment.yaml and service.yaml, changes in build-and-deploy.yml

* test deploy products

* change dockerfile

* test2345

* test new ecr

* change name tag

* test service url

* create service-url.txt

* new github token

* test12

* test service url

* test

* test5

* test4

* test

* test

* test 2

* test 45

* test 46

* test

* test 47

* test 567

* test 565

* test

* last changes in files

* add sleep

---------



* remove Maven CI workflow file

* rename API testing workflow from 'api orders testing' to 'api products testing'

* update API testing workflow to set dynamic branch environment variable and adjust service references; modify Dockerfile base image; enhance Postman collection with additional tests

* remove redundant tests for response body properties in Postman collection

* remove response time test from API Products Postman collection

* add newline at end of Dockerfile and ensure proper CMD execution

* update branch identification comment in API testing workflow

* test

* fix api testing

* Create test

* Update build-and-deploy.yml

Fix service URL.

---------
